Tests/InkTests/CodeTests.swift

```diff
@@ -21,7 +21,7 @@ final class CodeTests: XCTestCase {
         ```
         """)
 
-        XCTAssertEqual(html, "<pre><code>code()\nblock()</code></pre>")
+        XCTAssertEqual(html, "<pre><code>code()\nblock()\n</code></pre>")
```

Comment:
Is this really a part of the CommonMark spec? Seems like a regression.

Reply:
Look at example 94[https://spec.commonmark.org/0.29/#fenced-code-blocks](https://spec.commonmark.org/0.29/#fenced-code-blocks)

They have returns on last line on fenced blocks. I don't know the history.